component-library/src/app/pages/input-documentation/input-doc-code.component.ts

```diff
@@ -255,10 +255,11 @@ inputConfig: IInputComponentConfig = ${stringify(this.inputConfigCodeView)}`
     return {
       ...this.inputConfig,
       size: value['size'].toLowerCase(),
-      hint: value['hint'] === 'True' ? 'Hint text' : '',
+      hint: value['hint'] === 'True' ? 'Hint text' : undefined,
```

Comment:
@Naseer-007 setting value to `undefined` will automatically hide it from object to string output